CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -269,136 +296,247 @@ local function createImageButtonWithText(name, position, image, imageDown, text,
 	return imageButton
 end
 
+local function createImageButtonWithText3D(name, position, image, imageDown, text, font, fontSize)
+	local imageButton = createImageButtonWithText(name, position, image, imageDown, text, font, fontSize)
+	imageButton.Size = UDim2.new(0.5, -20, 0, 75)
+	imageButton.ScaleType = Enum.ScaleType.Slice
+	imageButton.SliceCenter = Rect.new(5, 5, 59, 59)
```

Comment:
it does seem if you are going to create a sliced image button, the slice should be passed in